client/modules/IDE/pages/IDEView.jsx

```diff
@@ -156,6 +157,10 @@ class IDEView extends React.Component {
     } else if (e.keyCode === 49 && ((e.metaKey && this.isMac) || (e.ctrlKey && !this.isMac)) && e.shiftKey) {
       e.preventDefault();
       this.props.setAllAccessibleOutput(true);
+      // 51 === 3
+    } else if (e.keyCode === 51 && ((e.metaKey && this.isMac) || (e.ctrlKey && !this.isMac)) && e.shiftKey) {
```

Comment:
you are using Ctrl+Shift+3 for creating new file looks weird, I think it should be something like Ctrl+N or Ctrl+Shift+N , @andrewn @catarak what's your opinion on this.